.github/workflows/continuous-integration.yml

```
@@ -7,7 +7,69 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.4'
+          tools: composer:v2
+
+      - name: Install XHProf (Windows)
+        shell: wsh
```

actionlint (1.7.7)

shell name "wsh" is invalid. available names are "bash", "cmd", "powershell", "pwsh", "python"

(shell-name)